### DISCUSSION TITLE
Cooking a grenade icon

### DISCUSSION THREAD

**Discussion by u1ps** [etlegacy/etlegacy#2104] [Ideas]
What about displaying icon above a teammate who is cooking a grenade. When there are a lot of players it gets hard to keep track of what is happening. Such icon would be good notification to teammates.

**Comment by Aranud** [comment]
With the newly implemented voice chat icon for "Fire in The hole" it would help a bit more about the player intention. 
Additionally, the player model animation while he cooking a grenade should also help.

I'm unsure if adding one additional information (like a counter) on top of player head will help ... 
With a few player certainly, each will not send grenades at the same time.
On server full (48 - 64) on goldrush near the tank at first axis spawn ... I don't even imagine the mess on the UI.

Perhaps I'm wrong, still it is possible to do so.

For record, the player state `grenadeTimeLeft` is what we are looking to track the remaining time before explosion.

**Reply by u1ps** [reply]
Just an idea no idea how it would look in real game, big mess or actually help. smaller icon?, turn on/off feature
